packages/design-system-docs/src/scripts/components/ReactPropDocs.jsx

```diff
@@ -28,11 +28,8 @@ class ReactPropDocs extends React.PureComponent {
 
   render() {
     return [
-      <h3 key="propDocsHeader">Props</h3>,
       <Table key="propDocsTable" stackBreakpoint="sm" scrollable>
-        <TableCaption className="ds-u-visibility--screen-reader">
-          React Properties Documentation
-        </TableCaption>
+        <TableCaption className="ds-u-padding-y--2 ds-u-font-size--h3">Props</TableCaption>
```

Comment:
Instead of using `ds-u-font-size--h3`, can we just put the h3 inside the caption

Reply:
Actually, can we just move the `h3` out of the caption altogether? Putting it in the caption means it will scroll with the table. Even if we accept that the caption notice scrolls with the table, I don't think its necessary for the heading to as well